src/components/ButtonGroup/ButtonGroup.test.jsx

```diff
@@ -0,0 +1,30 @@
+import { fireEvent, render } from '@testing-library/react';
+import ButtonGroup from './ButtonGroup';
+
+const ButtonGroupHelper = ({ handleSomething }) => (
+  <ButtonGroup
+    items={[
+      { id: 1, text: '버튼1' },
+    ]}
+    render={(item) => item.text}
+    keyOfItem={(item) => item.id}
+    onClick={handleSomething}
+  />
+);
+
+describe('ButtonGroup', () => {
+  it('전달한 목록과 렌더링 방식에 따라 내용을 출력한다.', () => {
+    const { getByText } = render(<ButtonGroupHelper />);
+
+    expect(getByText('버튼1')).not.toBeNull();
```

Comment:
```suggestion
    expect(getByText('버튼1')).toBeInTheDocument();
```

긍정문이 더 이해하기 쉬울 것 같아요